### PR TITLE
README.md: add a section about issue reporting

### PR DESCRIPTION
**Pull request by sruggier**
This is a followup to cirosantilli's comments on issue [#187](https://github.com/plasticboy/vim-markdown/issues/187).

By the way, GFM seems to no longer add line breaks for single newlines, so there doesn't seem to be any reason not to wrap text to less than 79 characters per line. However, to avoid breaking style with the rest of the README, I unwrapped the new text anyway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,20 @@ The following work on normal and visual modes:
 
 -   `:Tocv`: Same as `:Toc` for symmetry with `:Toch` and `Tocv`.
 
+## Issue reporting
+
+Issues are tracked within GitHub.
+
+When reporting issues, your report is more effective if you include a minimal example file that reproduces the problem. Try to trim out as much as possible, until you have the smallest possible file that still reproduces the issue. Paste the example inline into your issue report, quoted using four spaces at the beginning of each line, like this example from issue [#189](https://github.com/plasticboy/vim-markdown/issues/189):
```
Comment (cirosantilli) [inline README.md]: I'd shorten this to:

Provide a minimal buffer test case if applicable. Try to trim out as much as possible, until you have the smallest possible file that still reproduces the issue.

Include the actual buffer text, not just an image, as it makes it easier for others to reproduce the problem.

File: README.md [hunk 2]
```diff
@@ -162,6 +162,20 @@ The following work on normal and visual modes:
...
+
+```
+Minimal example:
+
+    ```
+    =
+    ```
+    bad!
+```
```
Comment (cirosantilli) [inline README.md]: I'd remove this example. I think it's clear from the above.

Comment (sruggier) [inline README.md]: For a user filing a bug report, it's not necessarily obvious how to quote the markdown to achieve the effect you did in #189. It took me a few minutes to figure it out just to make that example. I included the extra detail and the example to save users the trouble of discovering the right way to do it on their own. Shortening the text and removing the example would remove this detail, and make it less likely that a typical bug reporter is going to bother to include an example in a format convenient for you. Since you probably prefer clear bug reports, and want to save your users time, I'd recommend leaving it in there.

Comment (cirosantilli) [inline README.md]: OK, if you think this really helps, let's merge it. Can you move this section into CONTRIBUTING.md ? It's the standard GitHub location for this kind of thing.

**Comment by sruggier** [comment]
My mistake, I don't know why I didn't put it there in the first place :) I've rebased on master, made that change, and then force pushed the branch.

**Comment by cirosantilli** [comment]
Merged! Thanks for the PR sruggier.
